manifests/e/Eraser/Eraser/6.2.0.2992/Eraser.Eraser.locale.en-US.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate
+
+PackageIdentifier: Eraser.Eraser
+PackageVersion: 6.2.0.2992

```

Comment:
```suggestion
PackageVersion: 6.2.2992
```
![image](https://user-images.githubusercontent.com/11600822/120930476-47d6c500-c6e5-11eb-8fe5-9e2c201f1ed4.png)

Looks like it still hasn't been resolved yet; make sure to click the "Commit suggestion" button.